src/rust/organization-management/rebuild-sqlx.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+DBNAME=organization-management
+USER=postgres
+export POSTGRES_PASSWORD=testpassword
+CONTAINER_NAME=postgres-organization-management-compile
+
+cleanup() {
+    docker kill ${CONTAINER_NAME}
+}
+
+echo "Creating docker container:"
+
+docker run \
+    --detach \
+    --publish 5432:5432 \
+    --rm \
+    --name ${CONTAINER_NAME} \
+    --env POSTGRES_PASSWORD \
+    postgres
+
+trap cleanup EXIT INT
+
+echo ""
```

Comment:
What was this `echo` for? It looks like it can just be deleted, unless you want to have a message like `"Sleeping for 5 seconds to allow the database to come up"`.